src/pages/console/authz/permissions.mdx

```diff
@@ -16,26 +16,26 @@ You are going to define set up role-based authorization rules for each of the mo
 2. In the **Model** pane on the right, expand the **Owners** window. 
 3. Choose **Create**, **Read**, **Update** and **Delete** to specify that *Owners* have create, read, update, and delete access. The settings look as follows.
 
-![GSA](/images/console/10_ownersaccess.png)
+![Owner authorization rules on a GraphQL data model](/images/console/10_ownersaccess.png)
 
 ## To set a private authorization rule
 1. Using the *Books* data model that you created in the [Create a data model example](/console/data/data-model#Create-a-data-model-example), set the authorization mode to **Cognito user pool**. 
 2. In the **Model** pane on the right, expand the **Any signed-in users** window. 
 3. Choose **Create**, **Read**, and **Update** to specify that any signed-in authenticated user has create, read, and update, access. The settings look as follows.
 
-![GSA](/images/console/11_privatesaccess.png)
+![Private authorization rules on a GraphQL data model](/images/console/11_privatesaccess.png)
```

Comment:
Same here regarding the "The settings look as follows", does the image convey any extra info not mentioned in the paragraph above? If so we should write that out in the alt text, or write it out some more in the paragraph above.